intellij/src/main/kotlin/motif/intellij/MotifProjectService.kt

```diff
@@ -36,15 +37,17 @@ import com.intellij.psi.PsiClass
 import com.intellij.psi.PsiElement
 import com.intellij.ui.content.Content
 import com.intellij.ui.content.ContentFactory
+import javax.swing.Icon
 import motif.core.ResolvedGraph
-import motif.intellij.analytics.AnalyticsProjectComponent
+import motif.intellij.analytics.AnalyticsProjectService
 import motif.intellij.analytics.MotifAnalyticsActions
 import motif.intellij.ui.MotifErrorPanel
 import motif.intellij.ui.MotifScopePanel
 import motif.intellij.ui.MotifUsagePanel
 import org.jetbrains.kotlin.idea.KotlinLanguage
 
-class MotifProjectComponent(val project: Project) : ProjectComponent {
+@Service(Service.Level.PROJECT)
+class MotifProjectService(val project: Project) : Disposable {
```

Comment:
`MotifService`  should work better, the project is a leftover from ProjectComponent